src/main/java/org/jenkinsci/test/acceptance/po/Jenkins.java

```diff
@@ -112,7 +116,16 @@ public void restart() {
         visit("restart");
         clickButton("Yes");
 
-        waitFor(driver, not(hasContent("Please wait")), JenkinsController.STARTUP_TIMEOUT);
+        try {
+            waitFor(driver, not(hasContent("Please wait")), JenkinsController.STARTUP_TIMEOUT);
+            throw new TimeoutException();
+        }catch(TimeoutException e) {
```

Comment:
:ant: Strange code idiom. Assuming you really meant it to do what it seems to be doing, I think it would be clearer written thus:

```java
try {
    waitFor(driver, not(hasContent("Please wait")), JenkinsController.STARTUP_TIMEOUT);
catch (TimeoutException e) {/* ignore */}
visit(driver.getCurrentUrl());
waitFor(driver, hasElement(By.className("breadcrumbBarAnchor")), 10);
```